core.md

```diff
@@ -0,0 +1,127 @@
+# Governance of the NeoForged project
+
+The NeoForged project is split between [GitHub](https://github.com/NeoForged) and [Discord](https://discord.neoforged.net). It is a community of volunteers, maintaining a free and open source ecosystem of modifications to the Minecraft game.
+
+### What this document is
+
+This document is:
+ * A non-exhaustive list of the working principles of the project.
+ * Intended to help guide the project in times of uncertainty.
+
+This document is built on the fundamental principle of "people generally want to help, so we should not get in the way of it".
+
+The core ethic is "people over processes", so this document is intentionally light on processes - situations that come up are rarely one-size-fits-all, and should be treated on a case by case basis, with the best process to fit the person.
+
+### What this document is not
+
+This document is not:
+ * Intended to enforce rules on all members.
+ * Intended to catch all possibilities.
+ * Intended to prevent "rules lawyering".
+
+The Code of Conduct of the project is taken to be the content of [the Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), so this document is not the Code of Conduct. However, this may be altered with a copy held and maintained by the Project, only with overwhelming majority support by the entire team.
+
+Conflict resolution is a duty split between the Moderators and Steering Council; the former for interpersonal disputes in and with the community, and the latter for development-related disputes in the Project itself. Either team may wish to append a section to this document, but currently, this document is not conflict resolution.
+
+## Definitions
+
+Following are some core definitions that will be referred to in the later sections of the document.
+
+### The Project
+
+"The Project" refers to The NeoForged project, with a presence on both GitHub and Discord, as noted at the start of this document. It does not refer to any specific repository or subproject.
+
+### Subprojects
+
+Subprojects are large tasks that the Project itself is undergoing.
+
+These may be:
+
+* Repositories under the Project's GitHub.
+* Large, multi-repository tasks that are being undertaken (ie. updating the API to a new version of the Game).
+* Smaller, more targetted tasks that are being undertaken (ie. refactoring the Modloader or API to remove or replace or rework a feature).
+
```

Comment:
Consider clarifying that the maintainers and moderators team don't have to be odd as the council will break ties.

Reply:
They should usually not be asked for a full vote anyway, so I don't think it matters.